contracts/halo-pool/src/contract.rs

```diff
@@ -519,6 +942,144 @@ fn execute_update_pool_limit_per_user(
     Ok(res)
 }
 
+pub fn execute_add_phase(
+    deps: DepsMut,
+    env: Env,
+    info: MessageInfo,
+    new_start_time: u64,
```

Comment:
cái này không chỉ định lại whitelist cho nó à

Reply:
Supported updating whitelist address in add_phase fn